### PR TITLE
Refactor ImageDynamic

### PR DESCRIPTION
**Pull request by alan-hadyk**

### CODE REVIEW

File: src/components/atoms/IconDynamic/config.ts [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
-import Warning from "@app/../public/icons/dynamic/warning.svg";
-import Webpack from "@app/../public/icons/dynamic/webpack.svg";
+import Apollo from "@public/icons/dynamic/apollo.svg";
+import Chrome from "@public/icons/dynamic/chrome.svg";
+import CodeSandbox from "@public/icons/dynamic/codeSandbox.svg";
+import Download from "@public/icons/dynamic/download.svg";
+import ExternalLink from "@public/icons/dynamic/externalLink.svg";
+import Firefox from "@public/icons/dynamic/firefox.svg";
+import GitHub from "@public/icons/dynamic/gitHub.svg";
+import Graphql from "@public/icons/dynamic/graphql.svg";
```
Comment (angulawk) [inline src/components/atoms/IconDynamic/config.ts:8]: Choose one name to use it as a key and value - `GraphQL` or `Graphql`

File: src/components/atoms/ImageDynamic/ImageDynamic.tsx [hunk 2]
```diff
@@ -9,27 +9,11 @@ import React, { forwardRef } from "react";
...
-      isActive,
-      isHeightResponsive,
-      isResponsive,
-      shouldGlow,
-      shouldGlowOnHover,
       themeClasses,
+      variants,
     });
 
   const ImageComponent: SVGImage = imageDynamicComponents[imageName];
```
Comment (angulawk) [inline src/components/atoms/ImageDynamic/ImageDynamic.tsx:19]: consider removing ImageDynamin wrapper and think about IconDynamic component which looks exactly the same as ImageDynamic

File: src/components/atoms/IconDynamic/config.ts (hunk 1 above)
Comment (alan-hadyk) [reply src/components/atoms/IconDynamic/config.ts:8]: Changed to `GraphQL`

File: src/components/atoms/ImageDynamic/ImageDynamic.tsx (hunk 2 above)
Comment (alan-hadyk) [reply src/components/atoms/ImageDynamic/ImageDynamic.tsx:18]: Removed ImageDynamic wrapper, removed `IconDynamic` component (merged with `ImageDynamic`) and simplified `ImageDynamic`.

**Review by angulawk — APPROVED** [review]

**Pull request merged**